web/src/popover_menus.js

```diff
@@ -153,11 +153,30 @@ export function initialize() {
 
             instance.setContent(parse_html(render_left_sidebar_stream_setting_popover()));
             left_sidebar_stream_setting_popover_displayed = true;
+            //  When showing the popover menu, we want the
+            // "Add streams" tooltip and the "Filter streams"
+            //  tooltip to appear below the "Add streams" icon.
+            $("#add_streams_tooltip").get(0)._tippy.setProps({
+                placement: "bottom",
+            });
+            $("#filter_streams_tooltip").get(0)._tippy.setProps({
+                placement: "bottom",
+            });
             return true;
         },
         onHidden(instance) {
             instance.destroy();
             left_sidebar_stream_setting_popover_displayed = false;
+            //  After the popover menu is closed, we want the
+            //  "Add streams" tooltip and the "Filter streams"
+            //  tooltip to appear at it's original position that
+            //  is above the "Add streams" icon.
+            $("#add_streams_tooltip").get(0)._tippy.setProps({
+                placement: "top",
+            });
+            $("#filter_streams_tooltip").get(0)._tippy.setProps({
+                placement: "top",
+            });
         },
```

Comment:
This is wrong, since it would throw an error when `$("#add_streams_tooltip").get(0)._tippy` or `$("#filter_streams_tooltip").get(0)._tippy` doesn't exist. So, we can check for it and use `data-tippy-placement` to set placement for those tooltips in that case.